app/controllers/races/randomizers_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Races::RandomizersController < Races::ApplicationController
+  def show
+    render 'races/show'
+  end
+
+  def update
+    render :unauthorized unless @race.owner == current_user
+
+    @race.files.attach(params[:randomizer][:files])
+    Api::V1::RaceChannel.broadcast_to(@race, Api::V1::WebsocketMessageBlueprint.render_as_hash(
+      Api::V1::WebsocketMessage.new('new_attachment', {
+        message: 'The race owner has attached a new file',
+        race: Api::V4::RaceBlueprint.render_as_hash(@race, view: @race.type),
+        attachments_html: ApplicationController.render(partial: 'races/attachments', locals: {race: @race})
+      })
+    ))
+  end
```

Comment:
My idea had been to put everything related to races over the websocket just so that it was clear where to go for it.  I also wanted to force 3rd parties to use the websockets to make an integration so that the integration would be better user experience, if that makes sense?

I'm having a bit of hard time trying to put my thoughts into text that is somewhat coherent.  If you want I can dump my thoughts in a call.